Goji slack post

tommygoji opened the pull request:
Dear Ishikawa-kun,

Please pull, modify and merge the following files as you like:
result.html
js/result.js

I made sure they worked. Please be aware that you need to change its token. Instructions are written there.
![2017-06-24 15 27 45](https://user-images.githubusercontent.com/26486087/27506546-e0deaf1e-58f4-11e7-9754-bfb1f439a32e.png)
![2017-06-24 15 44 07](https://user-images.githubusercontent.com/26486087/27506548-e610c95e-58f4-11e7-8816-d3638c359921.png)

I suppose their UI's do not look professional, so please brush them up so they fit your standard.  Let me know. 

Thanks a lot as always for your leadership for our team.

Tommy Goji

tommygoji committed:
first trial to reflect slack_post

tommygoji committed:
Weekend work for integration of Slack Post function

AkihikoIshikawa commented:
1. HTML構造について
result.htmlに
headタグ、titleタグに被りがあるので、修正してください。

2. onButtonClick()について
result.htmlではなく、result.jsに書いたほうが良いと思います。
可読性を高めるためです。

3. formについて
ページ上にフォームを新たに作るのではなく、
`<form id="ad-face">`
に２つテキストボックスがあるので、
そこに加える形でお願いできますでしょうか。

nameに関しては、index.htmlで入力させるので、
その値を取得する形で考えています。（ので、テキストボックスは2つで良いかと。）

また、「confirm」を押す必要があるのは、
ユーザービリティが下がるので、
「submit」ボタン（整形カウンセリングを申し込む）ボタンだけ
用意すればよいかと思います。

4. tokenについて
「token.js」を創り、.gitignoreにtoken.jsを追加してくれませんか？
token.jsには、
`var token ='xoxp-190466...`
を入れ、result.jsでその変数（token）を呼び出す形が良いかと。

そして、token.jsをSlack上でやり取りしましょう。
現状の仕様だと、（deployする時に--を-に差し替える仕様だと、）
差し替え忘れや、deploy次のミスの原因となるため、
避けるべき運用とかんがえられるためです。

tommygoji committed:
Goji's Revision on Wed, June 28

Goji's revisions for Ishikawa-kun's review on Mon, June 26. Solved Ishikawa-kun's Comments 1, 2 and 4. Comment 3 is unresolved, which seems beyond Goji's current skill set.

tommygoji commented:
Dear Ishikawa-Kun,
Solved 1,2 & 4 of your comments. Could not solve your comment 3.
Please have a look at my post at Slack #general.
Thank you.
Tomotaka Goji

AkihikoIshikawa committed:
fix

AkihikoIshikawa committed:
fix conflict
